src/main/java/org/broadinstitute/hellbender/utils/pileup/ReadPileup.java

```diff
@@ -255,6 +265,13 @@ public int getNumberOfElements(final Predicate<PileupElement> peFilter){
         return (int)getElementStream().filter(peFilter).count();
     }
 
+	/**
+     * Shortcut for get the number of deletions
+     */
+    public int getNumberOfDeletions() {
```

Comment:
I've just add it as a shortcut, because I'm also using it instead of using the lambda expression. For API users it would be nice, but I could remove it if you think that is not going to be useful.